perun-wui-core/src/main/java/cz/metacentrum/perun/wui/model/beans/Author.java

```diff
@@ -59,6 +87,16 @@ public final int getPublicationsCount() {
 		return JsUtils.getNativePropertyArray(this, "authorships").length();
 	}
 
+	/**
+	 * Authorships related to this author.
+	 *
+	 * @param authorships List<Authorship> related to this author
+	 */
+	public final native void setAuthorships(List<Authorship> authorships)/*-{
+    	this.authorships = authorships;
```

Comment:
I think this should be done the same way as setAttributes(List<Attribute> attributes)